### DISCUSSION TITLE
Bem vindo

### DISCUSSION THREAD

**Discussion by rafaelaroca** [BIPES/BIPES#80] [Português] [answered]
Esta categoria do fórum de discussões é focada para usuários que falam português. Fique à vontade para dar sugestões, fazer perguntas e discutir sobre o uso do BIPES!

**Comment by williamcostaesilva** [comment]
Olá, parabéns pelo projeto, pela iniciativa.
Não conhecia o BIPES mas fiquei muito interessado em conhecer e praticar. 

Att

**Reply by williamcostaesilva** [reply]
Professor Rafael,
Como dito anteriormente, fiquei muito interessado em aprender a usar o BIPES. Com o ebook "**Uma introdução à Internet das Coisas e Sistemas Embarcados utilizando programação por blocos com BIPES e ESP8266 / ESP32**" eu consigo começar a usar o BIPES com a placa ESP32 ou ESP8266? Há algum outro material além do ebook como vídeo tutoriais etc que poderia indicar.

**Comment by rafaelaroca** [comment, marked as answer]
Oi @williamcostaesilva , que bom que você gostou! Fique à vontade para testar, e qualquer coisa estamos à disposição para ajudar. Abraços!
Rafael

**Reply by williamcostaesilva** [reply]
Professor Rafael,
Como dito anteriormente, fiquei muito interessado em aprender a usar o BIPES. Com o ebook "Uma introdução à Internet das Coisas e Sistemas Embarcados utilizando programação por blocos com BIPES e ESP8266 / ESP32" eu consigo começar a usar o BIPES com a placa ESP32 ou ESP8266? Há algum outro material além do ebook como vídeo tutoriais etc que poderia indicar.

**Comment by rafaelaroca** [comment]
Oi @williamcostaesilva, sim! O ebook tem todos os passos, desde o básico para vc iniciar o uso do BIPES sem precisar instalar nem configuração nenhum software em seu PC.

Em um resumo rápido:

1. Acesse https://bipes.net.br/flash/esp-web-tools/, conecte sua placa ESP e instale o MicroPython pela web.
2. Acesse https://bipes.net.br/ide/ clique em conectar e comece a testar!

Qualquer coisa nos avise.
abraços!

Rafael

**Comment by williamcostaesilva** [comment]
Muito obrigado pelo retorno professor.

Att, 
Professor Doutor William Costa e Silva

**Comment by triatebr** [comment]
Parabéns pela iniciativa .... comunidade tech / acadêmica agradece

att,

**Comment by zimba64** [comment]
por favor 
eu nao concigo jogar o programa para funcionar direto no esp 
ja tentei de achar como mas nao tem 
no video de apresentacao justamente neste ponto o o rafae teve o problema e deichou pra depois 
eu sou dislexico autodidata em eletronica e trabalho com eletronica a mais de 40 anos 
mas a dislexia atrapalhou muito . 
poderia me dar este passo a passo .

**Comment by JulioAprigio8** [comment]
Olá @rafaelaroca, conheci recentemente a plataforma do BIPES e achei bem interessante, vi no site que tem suporte para as placas BluePill e BlackPill mas não encontrei por la como configurar o firmware na placa, saberia me informar se tem algum documento para configurar essas placas?

**Reply by sk8board** [reply]
@JulioAprigio8 BIPES requires the use of MicroPython firmware for the microcontroller to function with BIPES. I do not see an official version of MicroPython firmware for the Bluepill or Blackpill. Therefore, I would not expect BIPES to work with Bluepill or Blackpill. https://micropython.org/download/

BIPES has been in beta version for quite some time. https://github.com/BIPES/BIPES/releases There are many "place holders" of code within BIPES that are there for structure for future development. 

Since MicroPython does not provide firmware for Bluepill and Blackpill along with BIPES provides the appearance of support for Bluepill and Blackpill, I suspect the BIPES team inserted "place holders" of code within BIPES for structure of future development, which resulted with the appearance of support for Bluepill and Blackpill. 

If you want have a good experience with BIPES, then I recommend to use a Raspberry Pi Pico, as BIPES has most development for the Pico. https://bipes.net.br/pico/ui/

If you prefer to use ESP32, then I recommend to check-out  https://github.com/QuirkyCort/IoTy
